eventActions/accountabilityActions.js

```diff
@@ -84,7 +84,8 @@ class accountabilityActions {
 			let hasPinnedMessage = false;
 
 			// Get the pinned messages within a channel
-			await currentChannel.fetchPinnedMessages().then(fetchedPins => {
+			//currentChannel.messages.fetchPinned()
```

Comment:
Did you leave this comment in on accident?

Reply:
Yes... removing it when I get home and pushing a commit to the PR

Reply:
No worries! Just wanted to confirm.